js/api.js

```diff
@@ -0,0 +1,20 @@
+const URL = 'https://28.javascript.pages.academy/kekstagram';
+
+const load = async () =>
+  fetch(`${URL}/data`)
+    .then((response) => response.json())
+    .catch((reject)=> reject());
```

Comment:
В catch не будет reject, туда как раз попадает ошибка, в нашем случае тут будет Error с нашим сообщением 'Данные не загрузились' это сообщение мы и покажем в нашем оповещении:
```suggestion
    .catch((reason)=> showAlert(reason.message));
```